### PR TITLE
add watchdog to WS test that's timing out

### PR DESCRIPTION
**Pull request by nickdesaulniers**
let's see what else times out now, and I'll add subsequent patches.

### CODE REVIEW

**Comment by nickdesaulniers** [comment]
This is in the spirit of "test failures are preferable to test timeouts, but this is merely a band-aid until we figure out what's the root cause."

**Comment by nickdesaulniers** [comment]
omg does the watchdog timer even work. wtf

**Comment by nickdesaulniers** [comment]
or, maybe I added it to the wrong test. ugh doofus :anger:

File: test/ws.js [hunk 1]
```diff
@@ -145,6 +160,8 @@ test('ws socket bus', function (t) {
 
         // Tally messages from other buses.
         bus.on('data', function (msg) {
+          watchdog.reset();
```
Comment (reqshark) [inline test/ws.js]: why reset here?

**Comment by reqshark** [comment]
haha ur comments are funny

File: test/ws.js (hunk 1 above)
Comment (nickdesaulniers) [inline test/ws.js]: for this library, `reset` is akin to petting the watchdog.  It does reset the internal timer.  If the timer ever falls to 0, :boom:

**Comment by reqshark** [comment]
@nickdesaulniers, you don't like `skip()` in the other PR because it just skips the questionable test instead of running it. understandable.

So now what's the objective here? Instead of `skip()`, you basically want `retry()`? Is that right?

**Pull request closed without merging**